engine/lite/components/kafka-tests/src/test/scala/pl/touk/nussknacker/engine/lite/components/LiteKafaUniversaJsonFunctionalTest.scala

```diff
@@ -95,6 +99,47 @@ class LiteKafaUniversaJsonFunctionalTest extends AnyFunSuite with Matchers with
     }
   }
 
+  test("sink with schema with additionalProperties: true/{schema}") {
+    val lax = List(ValidationMode.lax)
```

Comment:
What about of situation object with properties and `additionalProperties` on ? It also should be possible :) I can't find any examples of it.

Reply:
added 3 tests